addons/udes_stock/tests/test_picking_type.py

```diff
@@ -630,3 +630,108 @@ def test19_enforce_with_empty_preprocessed_location(self):
         picking.move_line_ids.write({
             'location_dest_id': self.test_location_02.id
         })
+
+    def test20_assert_multi_lot_picking_validation_prevented(self):
```

Comment:
This seems like two tests. One to check it works OK with one lot. One to check it fails if there's more than one.

Reply:
Good point, will split them out.

Reply:
This test has now been split into two. :)